src/main/java/frc/team2412/robot/subsystem/IntakeSubsystem.java

```diff
@@ -58,12 +73,16 @@ private IntakeSolenoidState(DoubleSolenoid.Value value) {
 
     // Define Hardware
 
+    @Log.MotorController(tabName = "IntakeSubsystem")
     private final WPI_TalonFX motorOuterAxle;
+    @Log.MotorController(tabName = "IntakeSubsystem")
     private final WPI_TalonFX motorInnerAxle;
 
-    private final DoubleSolenoid solenoid;
+    @Log
+    public int value = 1;
```

Comment:
What's this for?

Reply:
It was used to test logging in the simulation, we have now deleted it.